README.md

```diff
@@ -125,7 +125,7 @@ And we're back! Use this repo to share and keep track of software, tech, CS, PM,
 | [TransMarket Group](https://boards.greenhouse.io/transmarketgroup?gh_src=295a85457us) | Chicago, IL | [Algorithmic Trader Intern](https://boards.greenhouse.io/transmarketgroup/jobs/4023650007?t=295a85457us) <br/> [Quantitative Trader Intern](https://boards.greenhouse.io/transmarketgroup/jobs/4023644007?t=295a85457us) |
 | [Procter & Gamble](https://www.pgcareers.com/global/en/job/R000086643/IT-Engineer-Internship) | Cincinnati, OH | IT Engineer Intern (No Sponsorship Available) |
 | [Five Rings](https://fiverings.avature.net/careers) | NYC | [Software Dev Intern](https://fiverings.avature.net/careers/FolderDetail/New-York-New-York-United-States-Software-Developer-Summer-Intern-2024/649) <br/> [Quant Researcher Intern](https://fiverings.avature.net/careers/FolderDetail/New-York-New-York-United-States-Quantitative-Researcher-Summer-Intern-2024/645) <br/> [Quant Trader Intern](https://fiverings.avature.net/careers/FolderDetail/New-York-New-York-United-States-Quantitative-Trader-Summer-Intern-2024/643) <br/> (Winter 2024 also available) | 
-| [Milliman](https://recruiting2.ultipro.com/MIL1017/JobBoard/f54234e9-dfde-b183-fd20-4fbdb19cba7a/OpportunityDetail?opportunityId=07e563a5-18f6-4e9f-a719-452971adba46)| Omaha, NE | Software Engineer Intern |
+| [Milliman](https://recruiting2.ultipro.com/MIL1017/JobBoard/f54234e9-dfde-b183-fd20-4fbdb19cba7a/OpportunityDetail?opportunityId=07e563a5-18f6-4e9f-a719-452971adba46)| Omaha, NE <br/>  Windsor, CT | Software Engineer Intern <br/> [Healthcare Analytics Intern Spring 2024 (CMH Health)](https://www.linkedin.com/jobs/view/3666135380/?eBP=JYMBII_JOBS_HOME_ORGANIC&recommendedFlavor=ACTIVELY_HIRING_COMPANY&refId=rnKyo6MZN%2BDDkzq3b2Jziw%3D%3D&trackingId=x%2FyKzqB58ORjCaZUDKFijQ%3D%3D&trk=flagship3_jobs_discovery_jymbii)|
```

Comment:
As stated in the contributing guidelines, don't add links from third-party sites like LinkedIn. You should always go to the company's own career site first and contribute links from _there._

Reply:
Other issues:
- The location is wrong. If you go to the actual posting on Milliman's career site, you'll see that the available locations are different (this is why we say to always go to the source).
- This is a spring internship, not a summer internship, so this posting doesn't belong in this README anyway.